x/ratesync/types/types.go

```diff
@@ -0,0 +1,211 @@
+package types
+
+import (
+	"fmt"
+	"slices"
+	"strconv"
+	"strings"
+
+	"cosmossdk.io/errors"
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	icatypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/types"
+	host "github.com/cosmos/ibc-go/v7/modules/core/24-host"
+	ibcexported "github.com/cosmos/ibc-go/v7/modules/core/exported"
+	liquidstakeibctypes "github.com/persistenceOne/pstake-native/v2/x/liquidstakeibc/types"
+)
+
+func (hc HostChain) ValidateBasic() error {
+	err := host.ConnectionIdentifierValidator(hc.ConnectionId)
```

Comment:
check that no other host chain has the same connection id

Reply:
nope, duplication is allowed